core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -740,6 +741,7 @@ void populate1() {
       defineMethod(CONCAT_WS_SPARK,
           BuiltInMethod.MULTI_TYPE_STRING_ARRAY_CONCAT_WITH_SEPARATOR.method,
           NullPolicy.ARG0);
+      defineMethod(ORACLE_CONVERT, BuiltInMethod.ORACLE_CONVERT.method, NullPolicy.ARG0);
```

Comment:
What do you think about changing `ORACLE_CONVERT` to `CONVERT_ORACLE`? It looks like the current naming format is functionName + dbType, for example: CONCAT_WS_SPARK, BIT_COUNT_MYSQL.

Reply:
Sure, I'll change it later, as well as `oracleConvert` in SqlFunctions.